ui/preferences/src/main/res/layout/choose_home_screen_order_dialog_header.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="60dp"
```

Comment:
Does this need to have a hard-coded height? Depending on the language, the translations could need two lines. If you just use `wrap_content`, it always takes whatever height it needs

Reply:
True, that is better. I the height always wrap_content, but instead added padding where it was needed  👍